types/hooks/screens/useMessageHook.types.ts

```diff
@@ -21,6 +21,7 @@ export interface ReplyMessage {
   message_type: MessageType;
   updated_at: string;
   chatType?: 'ANONYMOUS' | 'SIGNED';
+  attachments?: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` property is typed as `any`. This is not a good practice as it defeats the purpose of TypeScript's static typing. It would be better to define an interface for attachments that includes all the properties you expect an attachment to have. This will make your code more robust and less prone to runtime errors.

```diff
-  attachments?: any;
+  attachments?: Attachment[];
```

Where `Attachment` could be something like:

```typescript
interface Attachment {
  id: string;
  type: 'image' | 'video' | 'audio' | 'file';
  url: string;
  // other properties...
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->